deploy/config/client/env.local.js

```diff
@@ -0,0 +1,19 @@
+window.env = {
```

Comment:
Please [install the pre-commit hooks](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/docs/developer/index.md) added recently. The pre-commit hook will convert the double quotes to single quotes.